samza-core/src/main/java/org/apache/samza/execution/ExecutionPlanner.java

```diff
@@ -57,7 +59,7 @@ public ExecutionPlanner(Config config, StreamManager streamManager) {
     this.streamManager = streamManager;
   }
 
-  public JobGraph plan(StreamGraph streamGraph) throws Exception {
+  public StreamPlan plan(StreamGraph streamGraph) throws Exception {
```

Comment:
I name it StreamPlan to be consistent with StreamApplication, StreamGraph, StreakTask .. :). In Flink it's called StreamingPlan so I guess I will keep it for now.